crypto/src/main/scala/coop/rchain/crypto/util/SecureRandomUtil.scala

```diff
@@ -0,0 +1,20 @@
+package coop.rchain.crypto.util
+
+import java.security.SecureRandom
+import scala.util.{Success, Try}
+
+object SecureRandomUtil {
+  private val instances = List(
+    "NativePRNGNonBlocking",
+    "Windows-PRNG",
+    "SHA1PRNG"
+  )
+
+  def getSecureRandomInstance: SecureRandom =
+    instances.iterator //use iterator for lazy evaluation
+      .map(name => Try(SecureRandom.getInstance(name)))
+      .collectFirst {
```

Comment:
This should be the same as find(_.isSuccess) which is a bit more readable

Reply:
`find(_.isSuccess)` would return an `Option[Try[SecureRandom]]` whereas the current version returns `Option[SecureRandom]` since we have already destructured the `Try`.

Reply:
Right. Never mind then.

Reply:
@birchmd I'm wondering if this shouldn't be a `lazy val`?

Reply:
Yes, I like that idea. I'll make that change

Reply:
Done